### PR TITLE
Update the jboss-javaee-web-6.0 version and the javaee6.web.spec.version...

### PR DESCRIPTION
**Pull request by sgilda**
... properties to 3.0.0.Beta1-redhat-1.

### CODE REVIEW

File: helloworld/pom.xml [hunk 1]
```diff
@@ -54,7 +54,7 @@
          <dependency>
             <groupId>org.jboss.spec</groupId>
             <artifactId>jboss-javaee-6.0</artifactId>
-            <version>${javaee6.web.spec.version}</version>
+            <version>3.0.0.Beta1-redhat-1</version>
```
Comment (pmuir) [inline helloworld/pom.xml:47]: I think you should also remove the version definition higher up here.

File: html5-mobile/pom.xml [hunk 2]
```diff
@@ -38,7 +38,7 @@
             <dependency>
                 <groupId>org.jboss.spec</groupId>
                 <artifactId>jboss-javaee-6.0</artifactId>
-                <version>${javaee6.spec.version}</version>
+                <version>3.0.0.Beta1-redhat-1</version>
```
Comment (pmuir) [inline html5-mobile/pom.xml:43]: We should keep the property based version here.

File: html5-mobile/pom.xml [hunk 3]
```diff
@@ -19,7 +19,7 @@
       <!-- Timestamp format for the maven.build.timestamp property -->
       <!-- You can reference property in pom.xml or filtered resources (must 
          enable third-party plugin if using Maven < 2.1) -->
-      <javaee6.spec.version>3.0.0.Beta1</javaee6.spec.version>
+      <javaee6.spec.version>3.0.0.Beta1-redhat-1</javaee6.spec.version>
       <!-- Alternatively, comment out the above line, and un-comment the line below to
```
Comment (pmuir) [inline html5-mobile/pom.xml:27]: Here, we need to change which property is commented, and the surrounding text to make sense.

File: kitchensink-ear/pom.xml [hunk 4]
```diff
@@ -58,7 +58,7 @@
         <dependency>
             <groupId>org.jboss.spec</groupId>
             <artifactId>jboss-javaee-6.0</artifactId>
-            <version>${javaee6.spec.version}</version>
```
Comment (pmuir) [inline kitchensink-ear/pom.xml:61]: See html5-mobile

File: kitchensink-jsp/pom.xml [hunk 5]
```diff
@@ -36,7 +36,7 @@
         <dependency>
             <groupId>org.jboss.spec</groupId>
             <artifactId>jboss-javaee-web-6.0</artifactId>
-            <version>${javaee6.web.spec.version}</version>
+            <version>3.0.0.Beta1-redhat-1</version>
```
Comment (pmuir) [inline kitchensink-jsp/pom.xml:37]: See html5-mobile

File: kitchensink/pom.xml [hunk 6]
```diff
@@ -37,7 +37,7 @@
         <dependency>
             <groupId>org.jboss.spec</groupId>
             <artifactId>jboss-javaee-6.0</artifactId>
-            <version>${javaee6.spec.version}</version>
+            <version>3.0.0.Beta1-redhat-1</version>
```
Comment (pmuir) [inline kitchensink/pom.xml:39]: See html5-mobile

File: numberguess/pom.xml [hunk 7]
```diff
@@ -56,7 +56,7 @@
          <dependency>
             <groupId>org.jboss.spec</groupId>
             <artifactId>jboss-javaee-6.0</artifactId>
-            <version>${javaee6.web.spec.version}</version>
+            <version>3.0.0.Beta1-redhat-1</version>
```
Comment (pmuir) [inline numberguess/pom.xml:49]: Need to remove property definition as well

File: servlet-async/pom.xml [hunk 8]
```diff
@@ -55,7 +55,7 @@
          <dependency>
             <groupId>org.jboss.spec</groupId>
             <artifactId>jboss-javaee-6.0</artifactId>
-            <version>${javaee6.web.spec.version}</version>
+            <version>3.0.0.Beta1-redhat-1</version>
```
Comment (pmuir) [inline servlet-async/pom.xml:48]: Need to remove property definition as well

File: servlet-filterlistener/pom.xml [hunk 9]
```diff
@@ -50,7 +50,7 @@
          <dependency>
             <groupId>org.jboss.spec</groupId>
             <artifactId>jboss-javaee-6.0</artifactId>
-            <version>${javaee6.web.spec.version}</version>
+            <version>3.0.0.Beta1-redhat-1</version>
```
Comment (pmuir) [inline servlet-filterlistener/pom.xml:43]: Need to remove property definition as well

File: greeter/pom.xml [hunk 10]
```diff
@@ -27,13 +27,6 @@
       <!-- Define the version of JBoss' Java EE 6 APIs we want to import. 
         Any dependencies from org.jboss.spec will have their version defined by this 
         BOM -->
```
Comment (pmuir) [inline greeter/pom.xml]: These three lines (27, 28, 29) should go as well.

File: helloworld-rs/pom.xml [hunk 11]
```diff
@@ -27,13 +27,6 @@
       <!-- Define the version of JBoss' Java EE 6 APIs we want to import. 
         Any dependencies from org.jboss.spec will have their version defined by this 
         BOM -->
```
Comment (pmuir) [inline helloworld-rs/pom.xml]: These three lines (27, 28, 29) should go as well.

File: helloworld/pom.xml [hunk 12]
```diff
@@ -27,13 +27,6 @@
       <!-- Define the version of JBoss' Java EE 6 APIs we want to import. 
         Any dependencies from org.jboss.spec will have their version defined by this 
         BOM -->
```
Comment (pmuir) [inline helloworld/pom.xml]: These three lines (27, 28, 29) should go as well.

File: html5-mobile/pom.xml [hunk 13]
```diff
@@ -19,14 +19,12 @@
...
-      <!-- 
-      <javaee6.spec.version>3.0.0.Beta1-redhat-1</javaee6.spec.version> 
-      -->
-   </properties>
+
+      <!--<javaee6.spec.version>3.0.0.Beta1-redhat-1</javaee6.spec.version>-->
+      <!-- Alternatively, uncomment the above line, and comment out the line below to
+      use version 3.0.0.Beta1 which is a release designed
+      to work with JBoss AS 7. It requires you have access to the JBoss.org maven repository. -->
+      <javaee6.spec.version>3.0.0.Beta1-redhat-1</javaee6.spec.version>     </properties>
```
Comment (pmuir) [inline html5-mobile/pom.xml]: THis wasn't the final version I went with...

File: html5-mobile/pom.xml (hunk 2 above)
Comment (pmuir) [inline html5-mobile/pom.xml:43]: We still need ot use the property here.

File: html5-mobile/pom.xml (hunk 13 above)
Comment (pmuir) [inline html5-mobile/pom.xml]:
```
  <javaee6.spec.version>3.0.0.Beta1-redhat-1</javaee6.spec.version>
  <!-- Alternatively, comment out the above line, and un-comment the line below to 
    use version 3.0.0.Beta1 which is a release design for JBoss AS 7 (community version).
    It requires you have access to the JBoss EAP 6 maven repository. -->
  <!-- 
  <javaee6.spec.version>3.0.0.Beta1</javaee6.spec.version> 
  -->
```

**Comment by pmuir** [comment]
Couple more changes needed :-)

**Comment by sgilda** [comment]
Pete, let me know know if this is what you were looking for or if I need to make any more changes.

**Comment by pmuir** [comment]
Merged.

**Comment by sgilda** [comment]
Hooray!

**Pull request closed without merging**